sdk/schemaregistry/ci.yml

```diff
@@ -0,0 +1,29 @@
+# DO NOT EDIT THIS FILE
+# This file is generated automatically and any changes will be lost.
+
+trigger:
+  branches:
+    include:
+      - master
+      - release/*
+      - hotfix/*
+  paths:
+    include:
+      - sdk/schemaregistry/
+
+pr:
+  branches:
```

Comment:
@mitchdenny The branches info here and in the above section are common to all Track 2 packages. Any reason why this needs to be repeated for all packages and cannot be in the parent yml file?